### PR TITLE
Fixes #1161 - Add title of item to title attribute of dropdown trigger

### LINKED ISSUE #1161: Action dropdown title
When action dropdown is associated to an item its title should be "**Name of item** Actions".

An example in a card:

image

image

In a list:

image

image

### PR DESCRIPTION
**Pull request by travisrcory**

### CODE REVIEW

**Comment by matuzalemsteles** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by travisrcory** [comment]
@matuzalemsteles updated!

File: packages/clay-list/src/ClayList.soy [hunk 1]
```diff
@@ -154,6 +154,9 @@
 					{param items: $item.actionItems /}
 					{param spritemap: $spritemap /}
 					{param triggerClasses: 'component-action' /}
+					{param triggerTitle kind="text"}
+						{$item.name}{sp}{msg desc=""}actions{/msg}
```
Comment (matuzalemsteles) [inline packages/clay-list/src/ClayList.soy]: This should be so, the map may change depending on the data.
```suggestion
						{$item[$fieldSchema.fieldsMap.title]}{sp}{msg desc=""}actions{/msg}
```

**Comment by matuzalemsteles** [comment]
Merged, thanks a lot @travisrcory!

**Pull request merged**